src/main/java/com/vhs/bts/services/LaptopService.java

```diff
@@ -21,6 +28,12 @@ public class LaptopService {
     private final ProcessorService processorService;
     private final ScreenService screenService;
 
+    @Scheduled(cron = "0 * * * * *")
+    public void scheduleTaskUsingCronExpression() {
+        log.info("Laptops available: \n{}", getLaptops().stream()
```

Comment:
We need a nice output of the logs.
We want to see just a map with an id and a name